packages/browser-utils/browser-utils.ts

```diff
@@ -0,0 +1,30 @@
+// See https://stackoverflow.com/a/58064481
+export const isIOS =
```

Comment:
Is there a need to export this?

Reply:
Currently used in tests to decide whether to skip or not.

Reply:
Removed the export.